docs/docs/gatsby-cli.md

```diff
@@ -66,121 +66,123 @@ gatsby new
+|      ओप्शंस      | विवरण                                                                              |
 | :--------------: | ---------------------------------------------------------------------------------------- |
-|  `-H`, `--host`  | Set host. Defaults to localhost                                                          |
-|  `-p`, `--port`  | Set port. Defaults to 9000                                                               |
-|  `-o`, `--open`  | Open the site in your (default) browser for you                                          |
-| `--prefix-paths` | Serve site with link paths prefixed (if built with pathPrefix in your gatsby-config.js). |
+`-H`, `--host` | होस्ट सेट करें। इसकी डिफ़ॉल्ट वैल्यू localhost है               |
+| `-p`, `--port`  | पोर्ट सेट करें। इसकी डिफ़ॉल्ट वैल्यू 9000 है         |                    |
+| `-o`, `--open`  | आपके लिए अपने (डिफ़ॉल्ट) ब्राउज़र में साइट खोलता है      |
+| `--prefix-paths` | लिंक उपसर्गों के साथ साइट सर्व करें (यदि आपके gatsby-config.js में pathPrefix के साथ बनाया गया है) |
 
-### `info`
+### `जानकारी`
 
-At the root of a Gatsby site, get helpful environment information which will be required when reporting a bug:
+एक Gatsby साइट की डिफॉल्ट में, उपयोगी एन्वाइरन्मेंट की जानकारी प्राप्त करें जो बग की रिपोर्ट करते समय आवश्यक होगी:
 
 `gatsby info`
 
-#### Options
+#### ओप्शंस
 
-|       Option        | Description                                             |
+|       ओप्शंस        | विवरण                                             |
 | :-----------------: | ------------------------------------------------------- |
-| `-C`, `--clipboard` | Automagically copy environment information to clipboard |
+| `-C`, `--clipboard` | क्लिपबोर्ड पर एन्वाइरन्मेंट की जानकारी को स्वचालित रूप से कॉपी करें |
 
 ### `clean`
 
-At the root of a Gatsby site, wipe out the cache (`.cache` folder) and public directories:
+एक Gatsby साइट की डिफॉल्ट में, कैश को मिटा दें (`। Cache` फ़ोल्डर) और पब्लिक फोल्डर्स:
 
 `gatsby clean`
 
-This is useful as a last resort when your local project seems to have issues or content does not seem to be refreshing. Issues this may fix commonly include:
+यह अंतिम उपाय के रूप में उपयोगी है जब आपके लोकल प्रोजेक्ट में इश्यू हो या कॉंटेंट रेफ्रेश नहीं होती है। यह आमतौर पर ठीक हो सकता है:
 
-- Stale data, e.g. this file/resource/etc. isn't appearing
-- GraphQL error, e.g. this GraphQL resource should be present but is not
-- Dependency issues, e.g. invalid version, cryptic errors in console, etc.
-- Plugin issues, e.g. developing a local plugin and changes don't seem to be taking effect
+- स्टेट डेटा, उदाहरण के लिए यह file/resource/etc दिखाई नहीं देता है।
+- GraphQL एरर, उदाहरण के लिए यह GraphQL रीसोर्स मौजूद होना चाहिए लेकिन ऐसा नहीं है।
+- डिपेंडेन्सी इश्यूस उदाहरण के लिए अमान्य संस्करण, कंसोल में गुप्त एरर आदि।
+- प्लगिन मुद्दे, उदाहरण के लिए एक लोकल प्लगइन विकसित करना और परिवर्तन प्रभावी होना प्रतीत नहीं होता है।
```

Comment:
`परिवर्तन प्रभावी होना प्रतीत नहीं होता है` => `उसके कुछ भी बदलाव दिखाई ना देना`

how about this? or anyother suggestion you have?